instance.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_instance" "public" {
+  ami                         = "ami-08333bccc35d71140"
+  associate_public_ip_address = true
+  instance_type               = "t3.micro"
+  key_name                    = "Terraform_demo"
+  vpc_security_group_ids      = []
```

Comment:
you didn't specify any security group here. could you please associate the public security group here.

Reply:
I made the required changes.  Can you please check now?